Makefile

```diff
@@ -361,8 +361,8 @@ case.h
 	./compile commands.c
 
 compile: \
-make-compile warn-auto.sh systype
-	( cat warn-auto.sh; ./make-compile "`cat systype`" ) > \
```

Comment:
Was `systype` even being used in any way by `./make-compile` and `./make-load`? I can't see that it was.

Reply:
Oh, you explained this in the very first comment here.